styles/stylesheet.css

```diff
@@ -0,0 +1,376 @@
+}
+
+h3 {
+  color: #3b6860;
+  font: 2rem "Irish Grover", cursive;
+}
+.hidden {
+  display: none;
+}
+h4 {
+  color: #3b6860;
+  font: 1.6rem "Irish Grover", cursive;
+}
+h5 {
+  color: #3b6860;
+  font: 1.3rem "Irish Grover", cursive;
+  margin:1rem;
+}
+p {
+  font: 1rem/1.5rem Arial, Helvetica, sans-serif;
+}
+a:link, a:visited {
+  text-decoration: none;
+  font: 1rem Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  color: #274841;
+}
+a:hover, a:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+button, a.button {
+  display: inline-block;
+  padding: .3rem .4rem;
+  border: solid .2rem #3b6860;
+  border-radius: .2rem;
+  color: #274841;
+  background-color: #85bbb1;
+  text-align: center;
+  font-variant: small-caps;
+  font-weight: bold;
+  line-height: 1rem;
+  box-sizing: content-box;
+  height: 1rem;
+}
+a.button:hover, a.button:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+button:hover, button:active {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+a:hover h4, a:active h4 {
+  color: #ffffad;
+  text-shadow: .2rem .2rem .3rem #274841;
+}
+/* Header styling */
+header {
+  width: 100%;
+  height: auto;
+  background: url("../images/header-image.jpg");
+  padding: 0;
+  border: 0;
+  margin: 0;
+  display: grid;
+  grid-template-areas:
+    "title title cart"
+    "spacer spacer spacer"
+    "nav nav nav";
+  grid-gap: 0;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-auto-rows: minmax(3rem, auto);
+}
+.title {
+  grid-area: title;
+  padding: 2rem;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+h1 {
+ color: #ffffff;
+ font: 4.5rem "Irish Grover", cursive;
+ padding: 1rem;
+ margin: 0;
+ box-sizing: border-box;
+}
+h1::before {
+  display: inline-block;
+  content: url("../images/adoptadog-logo-small.svg");
+  width: 5rem;
+}
+h2 {
+  color: #85bbb1;
+  font: italic 1.75rem Arial, Helvetica, sans-serif;
+  padding: 1rem;
+  margin: 0;
+  box-sizing: border-box;
+}
+.cartinfo {
+  grid-area: cart;
+  margin-right: 0;
+  margin-left: auto;
+  max-width: 14rem;
+  max-height: 14rem;
+  padding: 2rem;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-end ;
+  align-items: flex-start;
+}
+.cart {
+  min-width: 2rem;
+  max-width: 4rem;
+  margin: 2rem 1rem 2rem 2rem;
+  border: 0;
+  padding: 0;
+}
+header  p {
+  padding: 2rem 2rem 2rem 1rem;
+  margin: 0;
+  color: #ffffff;
+}
+header a.button {
+  margin-right: 2rem;
+}
+header > nav {
+  display: block;
+  padding: .5rem .5rem .5rem 3rem;
+  grid-area: nav;
+  width: 100%;
+  background-color: #bfbfbf;
+}
+nav > ul {
+  list-style-type: none;
+}
+nav li {
+  display: inline-block;
+  padding: 1rem;
+}
+/* Main styling */
+main {
+  margin: 0;
+  border: 0;
+  padding: 4rem;
+}
+article {
+  display: block;
+  height: auto;
+  padding: 1rem;
+  margin-bottom: 3rem;
+  box-sizing: border-box;
+}
+article > img {
+  float: left;
```

Comment:
The one time I'd allow a flow. I still think that using flex might be a better decision.